plugins/main/server/routes/wazuh-elastic.ts

```diff
@@ -38,35 +38,6 @@ export function WazuhElasticRoutes(router: IRouter) {
       ctrl.getCurrentPlatform(context, request, response),
   );
 
-  router.get(
-    {
-      path: '/elastic/visualizations/{tab}/{pattern}',
-      validate: {
-        params: schema.object({
-          tab: schema.string(),
-          pattern: schema.string(),
-        }),
-      },
-    },
-    async (context, request, response) =>
-      ctrl.createVis(context, request, response),
-  );
-
-  router.post(
-    {
-      path: '/elastic/visualizations/{tab}/{pattern}',
-      validate: {
-        params: schema.object({
-          tab: schema.string(),
-          pattern: schema.string(),
-        }),
-        body: schema.any(),
-      },
-    },
-    async (context, request, response) =>
-      ctrl.createClusterVis(context, request, response),
-  );
-
```

Comment:
**nitpick**: if the endpoints are not used anymore and are removed, we should add a reference into the changelog.